metrics/tarantool/luajit.lua

```diff
@@ -13,6 +13,7 @@ end
 local function set_gauge(name, description, value, labels)
     local gauge = metrics.gauge(prefix_name(name), description)
     gauge:set(value, labels or {})
+    return gauge
 end
```

Comment:
This is weird. Use `utils.set_gauge`instead of copy-pasting the code.

Reply:
#277